pyevmasm/__main__.py

```diff
@@ -65,10 +67,24 @@ def main():
                 if buf_set <= hex_set:  # subset
                     buf = binascii.unhexlify(buf)
 
+        # replace all swarm hashes with \x00
+        if args.no_swarmhash:
+            while True: #detect multiple swarm hashes, about 700 contracts do
```

Comment:
We can simplify it by using `re.sub` (regex replace). Example:
```python
In [9]: re.sub(b'\x12..', b'\x00'*4, b'\x1234 abc \x1233 \x1234')
Out[9]: b'\x00\x00\x00\x00 abc \x00\x00\x00\x00 \x00\x00\x00\x00'
```

So here it should probably be (pls check this):
```python
buf = re.sub(b'\xa1ebzzr0X .{34}', b'\x00'*43, buf)
```